### PR TITLE
Add June .NET FR info

### PR DESCRIPTION
**Pull request by maririos**
Following new guidance for release notes:
https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change

### CODE REVIEW

File: releases/2020-06/dotnet.md [hunk 1]
```diff
@@ -65,16 +65,21 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 - Added spatial geometry types.
 - Added `BinaryData` type.
 
-### Form Recognizer [Changelog]()
+### Form Recognizer [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md#100-preview2-06-09-2020)
```
Comment (ShivangiReja) [inline releases/2020-06/dotnet.md:68]: [Installation Instructions](https://github.com/Azure/azure-sdk/blob/be69550630d172ba68696d2f83366d8461648201/releases/2020-06/dotnet.md#installation-instructions) and[ library release type](https://github.com/Azure/azure-sdk/blob/be69550630d172ba68696d2f83366d8461648201/releases/2020-06/dotnet.md#preview) is missing for FR.

Comment (ShivangiReja) [inline releases/2020-06/dotnet.md:68]: [Installation Instructions](https://github.com/Azure/azure-sdk/blob/be69550630d172ba68696d2f83366d8461648201/releases/2020-06/dotnet.md#installation-instructions) and[ library release type](https://github.com/Azure/azure-sdk/blob/be69550630d172ba68696d2f83366d8461648201/releases/2020-06/dotnet.md#preview) are missing for FR.

Comment (maririos) [reply releases/2020-06/dotnet.md:71]: Thank you!

File: releases/2020-06/dotnet.md [hunk 2]
```diff
@@ -65,16 +68,21 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
+- `USReceipt` and related types have been removed. Information about a `RecognizedReceipt` must now be extracted from its `RecognizedForm`.
+- Method `GetFormTrainingClient` is removed from `FormRecognizerClient` and `GetFormRecognizerClient` is added to `FormTrainingClient`.
+- Other method and property renaming detailed in changelog.
 
 #### New Features
- - New features
+- Support to copy a custom model from one Form Recognizer resource to another.
+- Authentication using azure-identity credentials now supported.
 
 #### Key Bug Fixes
```
Comment (kinelski) [inline releases/2020-06/dotnet.md:83]:
```
Custom form recognition with labels can now parse results from forms that do not contain all of the expected labels (#11821).
```
This fix (mentioned in the changelog) introduces a breaking change (returned fields now can be `null`). Is it worth mentioning it?

Comment (maririos) [reply releases/2020-06/dotnet.md:83]: discussed offline. Decided to leave it out

**Pull request merged**